test1.py

```diff
@@ -1,88 +1,188 @@
+                ingredient = random.choice(list_of_ingredients)
+                print(ingredient)
+                kcal += ingredient.get('energy') * (amount/100)
+                if kcal > 1750:
+                    print(kcal)
+                    break
+                add_item = login1.add_meal_item(meal.get('id'),ingredient.get('id'),amount)
+                print(add_item)
+
+        elif meal.get('plan') == nutrition.get('id') and nutrition.get('description') in ['Thursday', 'Friday', 'Saturday', 'Sunday']:
+            kcal= 0
+            while True:
+                amount = 100
+                ingredients_req = login1.get_req('ingredient')
+                list_of_ingredients = ingredients_req.get('results')
+                ingredient = random.choice(list_of_ingredients)
+                print(ingredient)
+                add_item = login1.add_meal_item(meal.get('id'), ingredient.get('id'),amount)
+                print(add_item)
+                kcal += ingredient.get('energy') * (amount/100)
+                if kcal > 1900:
+                    print(kcal)
+                    break
+
+# ex4
+name_of_workout = "Workout38"
+add_workout = login1.add_workout_day(name_of_workout,"This is workout10")
 print(add_workout)
-# create days for the added workout
-
-get_workout = login2.get_req('workout')
-for workout in get_workout['results']:
-    if workout['name'] == name_of_workout:
-        id1 = workout['id']
-        add_day1 = login2.add_day(workout['id'],"Push",1)
-        add_day2 = login2.add_day(workout['id'],"Pull",2)
-        add_day3 = login2.add_day(workout['id'],"Legs",3)
-
-        days = login2.get_req('day')
-        list_of_days = list()
-        for day in days["results"]:
-            if day["training"] == id1:
-                list_of_days.append(day["id"])
-
-        for id in list_of_days:
-            for i in range(6):
-                add_exercise_per_day = login2.add_exercise(id,5,2)
-
-
-
-
-
 
+exercises_get = login1.get_req('exercise')
+limit = exercises_get.get('count')
+exercises_get_total = login1.get_req('exercise',0,limit)
+list_of_exercises = exercises_get_total.get('results')
+
+chest_exercises_list = list()
+back_exercises_list = list()
+legs_exercises_list = list()
+abs_exercises_list = list()
+arms_exercises_list = list()
+sholders_exercises_list = list()
+calves_exercises_list = list()
+
+for exercise in list_of_exercises:
+    if exercise.get('category') == 11:
+        chest_exercises_list.append(exercise)
+    elif exercise.get('category') == 12:
+        back_exercises_list.append(exercise)
+    elif exercise.get('category') == 9:
+        legs_exercises_list.append(exercise)
+    elif exercise.get('category') == 13:
+        sholders_exercises_list.append(exercise)
+    elif exercise.get('category') == 8:
+        arms_exercises_list.append(exercise)
+    elif exercise.get('category') == 14:
+        calves_exercises_list.append(exercise)
+    else:
```

Comment:
Here you should also use an elif statement, verifying the 'category' value of the 'abs_exercises'. This is done in automation to avoid the following scenario:

Let's say abs_exercises have 'category' 100. Another exercise type is added after some time which has 'category' 200. In this case, both the abs and the newly added exercise will be added to the abs_exercises_list.